omero/developers/build-system.txt

```diff
@@ -5,157 +5,79 @@ Build System
-        |                                       The generated `ome.model.*` files contain Hibernate
-        |                                       annotations for object-relational mapping.
+        |-- model ............................ The model component is special in that it produces
+        |                                      a jar specific to your database choice: model-psql.jar
+        |                                      The generated `ome.model.*` files contain Hibernate
+        |                                      annotations for object-relational mapping.
         |
-        |-- blitz ............................. The blitz component also performs code generation
-        |    |                                  producing artifacts for Java, Python, and C++.
-        |    |
-        |    \ blitz_tools.py ................. OMERO-specific SCons Environment definition
-        |                                       and other build tools.
+        |-- blitz ............................ The blitz component also performs code generation
+        |    |                                 producing artifacts for Java, Python, and C++.
+        |    \ blitz_tools.py ................ OMERO-specific SCons Environment definition
+        |                                      and other build tools.
         |     
-        |--tools .............................. Other server-components with special build needs.
-        |    |--build.xml
+        |--tools ............................. Other server-components with special build needs.
+        |    |--build.xml .................... Build scripts
+        |    |
         |    \--<tool-name>
-        |         |--build.xml
-        |         `--ivy.xml
+        |         |--build.xml ............... Build file
+        |         \--ivy.xml ................. Jar dependencies
         |
-        \--antlib ............................. Special component which is not built, but referenced by the build
+        \--antlib ............................ Special component which is not built, but referenced by the build
             |
-            \--resources
-                |--global.xml
-                |--directories.xml
-                |--lifecycle.xml
-                \--depdendencies.xml
+            \--resources ..................... Build resources
+                |--global.xml ................ Global build properties
+                |--hibernate.xml
+                |--lifecycle.xml ............. Ivy-related targets
+                \--version.xml ............... Version properties
 
 .. note::
     User examples are explained under :doc:`/developers/GettingStarted`
 
-Each of the components can also be built directly. For example,
-
-::
-
-    ./build.py -f components/server/build.xml
-
-Code generation
-^^^^^^^^^^^^^^^
-
 Unfortunately, just the above snapshot of the code repository omits some
 of the most important code. Many MB of source code is generated both by
```

Comment:
It's not part of these chnages but I think 'many megabytes' is much more readable.